src/test/java/org/gridsuite/study/server/utils/SendInput.java

```diff
@@ -14,12 +14,15 @@
 import org.springframework.cloud.stream.binder.test.InputDestination;
 import org.springframework.messaging.support.MessageBuilder;
 
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+
 import static org.gridsuite.study.server.StudyConstants.HEADER_RECEIVER;
 import static org.gridsuite.study.server.StudyConstants.QUERY_PARAM_RECEIVER;
 
 /**
  * Class that implements an action we want to execute after mocking an API call
- * See 'Post-serve actions' in https://wiremock.org/docs/extending-wiremock/
+ * See 'Post-serve actions' in <a href="https://wiremock.org/docs/extending-wiremock/">...</a>
```

Comment:
Link is not done correctly

Reply:
I have not made this one, I don't know why it has been changed this way

Reply:
Maybe your formatter. Please fill the `...` or delete the `href`.

Reply:
fixed